### PR TITLE
[21] Contact Form Validation

### PR DESCRIPTION
**Pull request by kamena1994**
## Task: `created the contact validation`
contactValidation

- `created contact.js and modified contact.html and contact.css`

### CODE REVIEW

**Review by ToluAkin — CHANGES_REQUESTED** [review]
I think you should close the PR till the other one is merged. There are files in this pull request, same as the other one with changes which will cause a merge conflict. @kamena1994

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@kamena1994 This is PR is for contact form validation not signup

File: submissions/benjaminKamena/Dom/signUp.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
+// sign-up form validation
```
Comment (ToluAkin) [inline submissions/benjaminKamena/Dom/signUp.js:1]: @kamena1994 This file is not suppose to be in this PR

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Review by ToluAkin — CHANGES_REQUESTED** [review]
Merge required, email and full name validation into one then  merge subject and message in to one just like  you did for signup.js @kamena1994

**Review by adaugochi — CHANGES_REQUESTED** [review]
Almost all your functions do the same thing, which is checking if a field is empty. You didn't make a function to do what it is supposed to (which is to ensure code reusability). I will advise that you write a single function(maybe call it required) that handles that. the only difference is the field and the error message you intend to display. In that case, pass them as a parameter on the function.

**Pull request closed without merging**